src/vsg/io/read.cpp

```diff
@@ -95,7 +95,7 @@ PathObjects vsg::read(const Paths& filenames, ref_ptr<const Options> options)
         };
 
         // use latch to syncronize this thread with the file reading threads
-        ref_ptr<Latch> latch(new Latch(filenames.size()));
+        ref_ptr<Latch> latch(new Latch(static_cast<uint32_t>(filenames.size())));
```

Comment:
We've fixed this another way - changing the Latch to use an atomic_size_t.  Could you revert this commit and update to VSG master.